src/bitset.c

```diff
@@ -0,0 +1,407 @@
+#include <limits.h>
+#include <roaring/bitset/bitset.h>
+#include <roaring/portability.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* Create a new bitset. Return NULL in case of failure. */
+bitset_t *bitset_create() {
```

Comment:
It looks like for me at least, it already doesn't work on master, but I think this needs to be in a namespace if compiled as c++.

Reply:
Fixed.